applications/sdp/mspi/src/main.c

```diff
@@ -207,6 +230,57 @@ static void xfer_execute(struct mspi_xfer_packet xfer_packet)
 	nrf_vpr_clic_int_pending_set(NRF_VPRCLIC, VEVIF_IRQN(HRT_VEVIF_IDX_WRITE));
 }
 
+void prepare_and_read_data(struct mspi_xfer_packet xfer_packet, volatile uint8_t *buffer)
+{
+	NRFX_ASSERT(nrfe_mspi_dev_cfg.ce_num < ce_vios_count);
+	NRFX_ASSERT(nrfe_mspi_dev_cfg.io_mode < SUPPORTED_IO_MODES_COUNT);
+
+	nrf_vpr_csr_vio_config_t config;
+
+	xfer_params.counter_value = 4;
```

Comment:
Magic number.

Reply:
It was copied from TX path. I would leave it, it will be overwritten by PR with TX timing alignments anyway.